pkg/apis/planetscale/v2/etcdlockserver_types.go

```diff
@@ -188,6 +188,13 @@ type EtcdLockserverTemplate struct {
 
 	// PeerService can optionally be used to customize the etcd peer Service.
 	PeerService *ServiceOverrides `json:"peerService,omitempty"`
+
+	// Tolerations allow you to schedule pods onto nodes with matching taints.
+	// WARNING: kubernetes adds default tolerations for node.kubernetes.io/not-ready
+	// and node.kubernetes.io/unreachable, you'll need to include these or the pod will be
+	// endlessly attempt reconcile difference.
+	// +kubebuilder:validation:EmbeddedResource
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
```

Comment:
This LGTM for the most part. Will you please run `make generate` to update the generated docs?